.github/workflows/release-prep.yml

```diff
@@ -48,38 +60,48 @@ jobs:
           sed -i "s/minor: int = \([0-9]\+\)/minor: int = $MINOR/g" heat/core/version.py
           sed -i "s/micro: int = \([0-9]\+\)/micro: int = $MICRO/g" heat/core/version.py
           sed -i "s/extension: str = .*/extension: str = None/g" heat/core/version.py
+          
+          { echo -e "# v${MAJOR}.${MINOR}.${MICRO} - ${{github.event.inputs.title}}\n${{ steps.release_drafter.outputs.body}}\n"; cat CHANGELOG.md; } > tmp.md
+          mv tmp.md CHANGELOG.md
 
           # Git configuration with anonymous user and email
           git config --global user.email ""
           git config --global user.name "Heat Release Bot"
 
           # Commit the changes
-          git add heat/core/version.py
+          git add heat/core/version.py CHANGELOG.md
           git commit -m "Bump version to $VERSION"
 
           # Push the changes
           git push --set-upstream origin pre-release/${VERSION}
 
           # Create PR for release
           gh pr create \
-          --base release \
+          --base stable \
           --head ${{ env.PR_BRANCH }} \
           --title "Heat ${{ env.VERSION }} - Release" \
           --body "Pre-release branch for Heat ${{ env.VERSION }}.
-          Any release work should be done on this branch, and then merged into the release branch and main, following git-flow.
+          Any release work should be done on this branch, and then merged into the release branch and main, following git-flow guidelines.
```

Comment:
just for my understanding, should the pre-release target this (`pre-release/xxx`) branch? 

And should the changes in `pre-release/xxx` be merged into `main` with the backport label, or should they be manually merged into `stable`?